partshop-backend/src/main/java/hu/kosztaauto/partshop/service/RaktarService.java

```diff
@@ -0,0 +1,19 @@
+package hu.kosztaauto.partshop.service;
+
+import hu.kosztaauto.partshop.database.Raktar;
+import hu.kosztaauto.partshop.repository.RaktarRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class RaktarService {
+
+    @Autowired
+    private RaktarRepository raktarRepository;
+
+    public List<Raktar> getAllWarehouses() {
```

Comment:
shouldnt public methods be written in pascal case?

Reply:
I prefer camelCase for methods, no matter what scope they have.